src/UserService.ts

```diff
@@ -425,6 +440,32 @@ class UserService extends Base {
     return (json as unknown) as License;
   }
 
+  async getSubscriptionSeatOfflineActivationCode({ username, subscriptionSeatId, activationCodeData }: { username: string; subscriptionSeatId: string; activationCodeData: ActivationCodeData }): Promise<OfflineActivationCode | null> {
+    const { apiEndpointAddress, fetch } = this.lensPlatformClient;
+    const url = `${apiEndpointAddress}/users/${username}/subscription-seats/${subscriptionSeatId}/activation-code`;
+    const json = await throwExpected(
+      async () => fetch.post(url, activationCodeData),
+      {
+        404(error) {
+          const message = error?.body.message;
+
+          if (typeof message === "string") {
+            if (message.includes("User")) {
+              return new UserNameNotFoundException(username);
+            }
+          }
+
+          return new NotFoundException("Subscription seat subscriptionSeatId not found");
```

Comment:
Do we want to say `subscriptionSeatId` in the error message? We could use the actual `subscriptionSeatId` value instead.